packages/perseus-editor/src/components/util.ts

```diff
@@ -80,3 +80,13 @@ export function getDefaultFigureForType(type: LockedFigureType): LockedFigure {
             throw new UnreachableCaseError(type);
     }
 }
+
+export function lockedPointsEqual(
+    point1: LockedPointType,
+    point2: LockedPointType,
+) {
+    return (
+        point1.coord[0] === point2.coord[0] &&
+        point1.coord[1] === point2.coord[1]
+    );
```

Comment:
Why is it saying line 90 isn't covered by tests?

Reply:
Strange. I'm not sure. Codecov gets its coverage info from the test runs we upload, so somewhere that data must be wrong. :/ When I run `yarn coverage` locally and then `open coverage/lcov-report/index.html`, I can see that the line is covered.